vagrant/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+
+roles_path = ../roles
+hash_behaviour=merge
```

Comment:
Not sure if the `hash_behaviour` is a solution for #45 -- haven't played around with it, but the documentation seems to suggest that is what controls the behavior of how variables get merged together.